### PR TITLE
Fixed errcheck issues in server/channels/api4/file_test.go

### PR DESCRIPTION
**Pull request by Aryakoste**
#### Ticket Link

Fixes https://github.com/mattermost/mattermost/issues/28387
Jira https://mattermost.atlassian.net/browse/MM-60754
#### Release Note

```release-note
NONE
```

### CODE REVIEW

**Comment by Aryakoste** [comment]
@hanzei In this file line 731 will show shadow variable error as dicussed. How to proceed with this.

**Comment by Aryakoste** [comment]
Was gonna declare it as var just needed confirmation because below two variables tf and expected was also needed to convert into var for this error to fix. Thanks @hanzei I have made the changes.

**Comment by Aryakoste** [comment]
Is it ok if i use this approach in other files where i encounter such errors.

File: server/channels/api4/file_test.go [hunk 1]
```diff
@@ -1147,7 +1165,8 @@ func TestGetPublicFile(t *testing.T) {
 	require.NoError(t, err)
 	require.NoError(t, th.cleanupTestFile(fileInfo))
 
-	th.cleanupTestFile(info)
+	err = th.cleanupTestFile(info)
+	require.NoError(t, err)
```
Comment (hanzei) [inline server/channels/api4/file_test.go:1169]: This code cleans up the test files 2 times, leading to an error. Please remove one call to `cleanupTestFile`

**Review by hanzei — APPROVED** [review]
LGTM :+1:

**Review by pvev — APPROVED** [review]
LGTM! thanks for your contribution @Aryakoste 🚀

**Pull request merged**